src/java/fr/paris/lutece/portal/web/admin/AdminFeaturesPageJspBean.java

```diff
@@ -115,8 +115,9 @@ public abstract class AdminFeaturesPageJspBean implements Serializable
     public void init( HttpServletRequest request, String strRight ) throws AccessDeniedException
     {
         _user = AdminUserService.getAdminUser( request );
+        Right right = RightHome.findByPrimaryKey( strRight );
 
-        if ( !_user.checkRight( strRight ) )
+        if ( !_user.checkRight( strRight ) || right == null )
         {
             throw new AccessDeniedException( "User " + _user.getAccessCode( ) + " does not have " + strRight + " right." );
```

Comment:
it would be better to lauch a second AccessDeniedException with a specific message when the "strRight" does not exists